python_modules/dagster/dagster_tests/asset_defs_tests/partition_mapping_tests/test_asset_partition_mappings.py

```diff
@@ -561,3 +566,90 @@ def test_identity_partition_mapping():
         zx.empty_subset().with_partition_keys(["z", "x"]), xy
     )
     assert result.get_partition_keys() == set(["x"])
+
+
+def test_partition_mapping_with_outputs():
```

Comment:
I'm planning on removing this test before merge. it's here right now for debugging help so i can see what certain values on the AssetsDef and in the context should be